### ISSUE TITLE
`mysql_tzinfo_to_sql` not in PATH

### ISSUE THREAD

**Issue by jessestuart** [Shippable/support#3580]
### TL;DR
Our builds recently began failing as a result of `mysql_tzinfo_to_sql` not being defined in `PATH`. We first noticed this yesterday with [this build](https://app.shippable.com/github/TodayTix/platform/runs/29376/1/console). 

### Details
This appears to be similar in cause to https://github.com/Shippable/support/issues/2318; and in fact, @ric03uec's [suggestion](https://github.com/Shippable/support/issues/2318#issuecomment-191612869) to simply add `export PATH=$PATH:/usr/local/mysql/bin/` to our `ci` step seems to have resolved the issue for now. (I'm still waiting for [this build](https://app.shippable.com/github/TodayTix/platform/runs/29398/1/console) to complete, but it did successfully load the tzinfo data.)

If it helps with your debugging: I noticed that the failing build was running `drydock/u16javall:v5.5.1`, while a passing build a few days ago was running `drydock/u14javall:prod` (there have been no changes to our `shippable.yml` in the interim).

### Impact
Low, given @ric03uec's workaround. Still, it would be helpful to understand what went wrong here :)

Thanks all!

**Comment by ambarish2012** [comment]
@jessestuart we are investigating, it is possible we missed adding that path.

**Comment by jessestuart** [comment]
Hi again!
Any update on the investigation here?

**Comment by ambarish2012** [comment]
@jessestuart I confirm this is a bug and will let you know when the fix is rolled out in production. Please continue to use the workaround for now.

**Comment by jessestuart** [comment]
Sounds good — thanks @ambarish2012!

**Comment by ambarish2012** [comment]
@jessestuart the bug has been fixed.

**Comment by jessestuart** [comment]
Thanks @ambarish2012! 👍
